src/app/dashboard/components/dashboard-new-cases/dashboard-new-cases.component.html

```diff
@@ -1 +1,77 @@
-<p>dashboard-new-cases works!</p>
+<div class="dashboard-new-cases" *ngIf="newCases">
+  <h1>
+    <span class="nr-of-cases">{{ newCases.caseCount }}</span> New Cases
+  </h1>
+  <div class="cases-container">
+    <div class="left-part">
+      <div class="row grey">
```

Comment:
i think we should use better names then left-part, right-part, yellow, red etc
maybe instead of yellow use classification-suspect and instead of red classification-confirmed and so on

Reply:
Changed.